### PR TITLE
Updated metadata for suite executed only in PSI env

### PR DESCRIPTION
**Pull request by sunilangadi2**
Signed-off-by: Sunil Angadi <sangadi@redhat.com>

# Description
Fixes: RHCEPHQE-4473
To fix the issue https://github.com/red-hat-storage/cephci/issues/1592
In IBM environment, we don't have release.yaml file access hence this cron pipeline job suite is failing 
these suites can only be executed in psi-only environments
Hence updated metadata related to that suite.

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: pipeline/metadata/4.3.yaml [hunk 1]
```diff
@@ -503,12 +503,10 @@
   rhbuild: "4.3"
   inventory:
     openstack: "conf/inventory/rhel-7-latest.yaml"
-    ibmc: "conf/inventory/ibm-vpc-rhel-7-latest.yaml"
   metadata:
     - schedule
     - tier-2
     - openstack
-    - ibmc
```
Comment (manasagowri) [inline pipeline/metadata/4.3.yaml:511]: Please add another tag - openstack-only

**Review by manasagowri — CHANGES_REQUESTED** [review]
Also please check in 5.1.yaml as well there is a test suite for multipath upgrade. Please make that also as openstack-only.

File: pipeline/metadata/4.3.yaml (hunk 1 above)
Comment (sunilangadi2) [reply pipeline/metadata/4.3.yaml:511]: Done the changes.

**Comment by sunilangadi2** [comment]
Thanks @manasagowri for the review 
Addressed the comments please review it.

**Review by manasagowri — DISMISSED** [review]
LGTM

File: pipeline/metadata/4.3.yaml [hunk 2]
```diff
@@ -503,12 +503,11 @@
   rhbuild: "4.3"
   inventory:
     openstack: "conf/inventory/rhel-7-latest.yaml"
-    ibmc: "conf/inventory/ibm-vpc-rhel-7-latest.yaml"
   metadata:
     - schedule
     - tier-2
     - openstack
```
Comment (psathyan) [inline pipeline/metadata/4.3.yaml:509]: should we also remove `openstack`?

Comment (sunilangadi2) [reply pipeline/metadata/4.3.yaml]: Done.

**Comment by sunilangadi2** [comment]
Thanks @psathyan for the review, yes `openstack` becomes redundant when we have `openstack-only` value
Removed it, Please review it.

**Review by psathyan — APPROVED** [review]

**Pull request merged**